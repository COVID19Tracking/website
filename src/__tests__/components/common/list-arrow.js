import React from 'react'
import renderer from 'react-test-renderer'
import ListArrow from '../../../components/common/list-arrow'

describe('Components : Common: Get Involved', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ListArrow items={([<p>Sample text</p>], [<p>More text</p>])} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
