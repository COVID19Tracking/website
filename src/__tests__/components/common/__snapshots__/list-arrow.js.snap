// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Get Involved renders correctly 1`] = `
<ul
  className="listArrow"
>
  <li
    className="listArrowItem"
  >
    <p>
      More text
    </p>
    <span
      aria-hidden={true}
      className="listArrowIcon"
    >
      â†’
    </span>
  </li>
</ul>
`;
